ocimotel/repoclient.go

```diff
@@ -30,18 +35,52 @@ func NewOciRepo(ref *ociMotelReference) (r OciRepo, err error) {
 	if ref.port != -1 {
 		port = fmt.Sprintf("%d", ref.port)
 	}
-	r.url = fmt.Sprintf("http://%s:%s", server, port)
-	queryURI := fmt.Sprintf("%s/v2/", r.url)
-	client := &http.Client{}
-	resp, err := client.Get(queryURI)
-	if err != nil {
-		return r, err
+
+	insecureSkipVerify := (sys.DockerInsecureSkipTLSVerify == types.OptionalBoolTrue)
+	tlsClientConfig := &tls.Config{
+		InsecureSkipVerify: insecureSkipVerify,
 	}
-	defer resp.Body.Close()
-	if resp.StatusCode == 200 {
-		return r, nil
+
+	if sys.DockerCertPath != "" {
```

Comment:
Same here, seems like we should have our own?

Reply:
Same response as above. Also note, there is a slight quirk in how certs are loaded.
ca.crt end with '.crt' and client.cert and client.key are suffixed so.